src/core/virtual_sites/lb_inertialess_tracers.cpp

```diff
@@ -188,23 +177,20 @@ void CoupleIBMParticleToFluid(Particle *p) {
   }
 }
 
-/******************
-   GetIBMInterpolatedVelocity
-Very similar to the velocity interpolation done in standard Espresso, except
-that we add the f/2 contribution - only for CPU
-*******************/
-
+/** Calculate the LB fluid velocity at a particle position.
+ *  Very similar to the velocity interpolation done in standard Espresso,
+ *  except that we add the f/2 contribution, cf. @cite guo02a.
+ *  The fluid velocity is obtained by linear interpolation,
+ *  cf. Eq. (11) in @cite ahlrichs99a.
```

Comment:
This is maybe going too far, however there is quite some inconsistency concerning capitalization of mid-sentence abbreviations "Eq.", "Eqs." vs. "eq.", "eqs.".

Reply:
I'm now only using a capital `Eq.` at the start of a sentence.